ouroboros-consensus/test-storage/Test/Ouroboros/Storage/FS/StateMachine.hs

```diff
@@ -555,9 +575,13 @@ shrinker Model{..} (At cmd) =
           fp :: FsPath
           fp = resolvePathExpr knownPaths pe
 
-      Get      h n  -> At . Get      h <$> shrink n
-      Put      h bs -> At . Put      h <$> shrinkBytes bs
-      Truncate h n  -> At . Truncate h <$> shrink n
+      Get      h n   -> At . Get      h <$> shrink n
+      GetAt    h n o -> do
+        n' <- shrink n
+        o' <- shrink o
```

Comment:
If `n = 5` and `o = 5`, this won't include `n' = 5, o' = 4` or `n' = 4, o' = 5`.
So instead:
```haskell
GetAt h n o -> At <$>
  [GetAt h n o' | o' <- shrink o] <>
  [GetAt h n' o | n' <- shrink n]
```
(not type-checked, maybe some parentheses are needed)